.github/workflows/ci.yml

```diff
@@ -151,6 +151,7 @@ jobs:
           - arm64_32-apple-watchos
           - armv7-sony-vita-newlibeabihf
           - i686-unknown-linux-gnu
+          - i686-unknown-hurd-gnu
```

Comment:
Does is support x86_64, maybe ARM as well?

Reply:
x86_64 and arm64 are being supported, but that's not in upstream llvm etc. yet.